packages/fxa-event-broker/src/queueworker/queueworker.service.ts

```diff
@@ -89,14 +94,12 @@ export class QueueworkerService
       // Verify that the queue exists
       const queueParts = this.queueName.split('/');
       const queueName = queueParts[queueParts.length - 1];
-      const queues = await this.sqs.listQueues().promise();
+      const command = new ListQueuesCommand({});
+      const queues = await this.sqs.send(command);
 
       if (!queues.QueueUrls || !queues.QueueUrls.includes(queueName)) {
-        await this.sqs
-          .createQueue({
-            QueueName: queueName,
-          })
-          .promise();
+        const command = new CreateQueueCommand({ QueueName: queueName });
```

Comment:
The [docs](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/sqs/command/CreateQueueCommand/) state:
> After you create a queue, you must wait at least one second after the queue is created to be able to use the queue.

Do I need to explicitly wait 1s here in the code? What would that look like?

Reply:
Since this is in the `onApplicationBootstrap` function, I say it is probably fine to wait 1s.

Reply:
> Locally for tests, it runs faster, and its not expected that if we need to create these queues than a publisher is going to wait for this consumer, so waiting isn't needed.

Thanks! IIRC, the SQS publishers in FxA's case are GCP PubSub (for account/subscription-related events) and GCP Firestore (for ??), with `fxa-event-broker` as an SQS consumer -- is that right? So if anything is waiting 1s here, it's not the event broker, but PubSub or Firestore?